tinygrad/shape/shapetracker.py

```diff
@@ -3,8 +3,8 @@
 from enum import Enum, auto
 import functools
 from typing import Dict, Tuple, Union, List, Optional, Callable, cast, NamedTuple
-from tinygrad.helpers import prod, DEBUG
-from tinygrad.shape.symbolic import Variable, MulNode, NumNode, Node, SumNode, is_sym_int
+from tinygrad.helpers import prod, DEBUG, GlobalCounters
```

Comment:
Definitely no GlobalCounters here.

Reply:
ok. closing this. will put symbols somewhere in the middle